File: catalog/admin/char/delete.php
<?php
defined('AUTH') or die('Restricted access');

include_once $_SERVER['DOCUMENT_ROOT'].'/components/catalog/admin/classes/AdminCatalogChar.php';

$id = $SITE->d[5];

$CHAR = new AdminCatalogChar();
$catalog_id = $CHAR->delete($id);

header('location: /admin/com/catalog/char/'.$catalog_id);
exit;

File: dan/system/catalog/char/delete.py
import sys
sys.path.append('system/catalog/classes')
from system.catalog.classes.Char import Char


def delete(SITE):
    print('FUNCTION -> system-> calalog -> char -> delete')

    CHAR = Char(SITE)
    catalog_id = CHAR.deleteName(SITE.p[3])

    return {'redirect': '/system/catalog/char/' + str(catalog_id)}

